kroxylicious-api/pom.xml

```
@@ -18,24 +18,18 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
-    <artifactId>kroxylicious-filter-api</artifactId>
+    <artifactId>kroxylicious-api</artifactId>
     <version>0.3.0-SNAPSHOT</version>
     <packaging>jar</packaging>
 
     <properties>
         <libs.dir>libs</libs.dir>
     </properties>
 
-    <name>Kroxylicious Filter API</name>
-    <description>The public interfaces, classes, etc required to author custom filters and configure Kroxylicious to use them</description>
+    <name>Kroxylicious API</name>
+    <description>generic APIs used throughout Kroxylicious</description>
```

Nit: not sure generic is a great term here.

```suggestion
    <description>The public interfaces, classes, etc required to build filters and interact programatically with Kroxylicious</description>
```

---

Yeah good idea to update it, this was the description from kroxylicious-api